include_core/omrport.h

```diff
@@ -1388,6 +1388,7 @@ typedef enum OMRProcessorArchitecture {
 	OMR_PROCESSOR_S390_Z14,
 	OMR_PROCESSOR_S390_Z15,
 	OMR_PROCESSOR_S390_Z16,
+	OMR_PROCESSOR_S390_Z17
```

Comment:
@dchopra001 Can you please add a comma after `OMR_PROCESSOR_S390_Z17`

Reply:
Missed that. Fixed now. Thanks!